app/src/main/res/layout-land/profile_chooser_profile_view.xml

```diff
@@ -21,69 +21,71 @@
   </data>
 
   <androidx.constraintlayout.widget.ConstraintLayout
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content">
+    android:id="@+id/profile_chooser_item"
+    android:layout_width="wrap_content"
+    android:layout_height="wrap_content"
+    android:layout_marginStart="16dp"
+    android:layout_marginEnd="16dp"
+    android:gravity="center_horizontal"
+    android:orientation="vertical"
+    app:layoutMarginBottom="@{hasProfileEverBeenAddedValue ? @dimen/space_16dp : @dimen/space_28dp}"
+    app:layoutMarginTop="@{hasProfileEverBeenAddedValue ? @dimen/space_16dp : @dimen/space_24dp}"
+    app:layout_constraintEnd_toEndOf="parent"
+    app:layout_constraintStart_toStartOf="parent"
+    app:layout_constraintTop_toTopOf="parent">
 
-    <LinearLayout
-      android:id="@+id/profile_chooser_item"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:layout_marginStart="16dp"
-      android:layout_marginEnd="16dp"
-      android:gravity="center_horizontal"
-      android:orientation="vertical"
-      app:layout_constraintEnd_toEndOf="parent"
+    <de.hdodenhof.circleimageview.CircleImageView
+      android:id="@+id/profile_avatar_img"
+      android:layout_width="72dp"
+      android:layout_height="72dp"
+      app:civ_border_color="@color/avatarBorder"
+      app:civ_border_width="1dp"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent"
-      app:layoutMarginBottom="@{hasProfileEverBeenAddedValue ? @dimen/space_16dp : @dimen/space_28dp}"
-      app:layoutMarginTop="@{hasProfileEverBeenAddedValue ? @dimen/space_16dp : @dimen/space_24dp}">
+      profile:src="@{viewModel.profile.avatar}" />
 
-      <de.hdodenhof.circleimageview.CircleImageView
-        android:id="@+id/profile_avatar_img"
-        android:layout_width="72dp"
-        android:layout_height="72dp"
-        android:layout_marginBottom="8dp"
```

Comment:
Not sure if its a valid comment but this (marginBottom) was missed or has been adjusted somwhere else.

Reply:
All the attributes for res/layout-land/profile_chooser_profile_view.xml  image are the same as mine.